doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_recovery.rst

```diff
@@ -0,0 +1,183 @@
+.. _ug_nrf54h20_suit_recovery:
+
+SUIT manufacturer application recovery
+######################################
```

Comment:
```suggestion
Configuring and programming SUIT manufacturer application recovery
##################################################################
```
Because this is not only a conceptual page, but  you have steps and procedures here.